packages/coverage-c8/src/index.ts

```diff
@@ -1,6 +1,11 @@
-export * from './takeCoverage'
+import * as coverage from './takeCoverage'
 
-export async function getProvider() {
```

Comment:
This removal of named export `getProvider` caused a breaking change. The `@vitest/coverage-istanbul` is still exporting it as both named and default. Was this intentional @sheremet-va? 

There are some reports where users are using different versions of `vitest` and `@vitest/coverage-c8` and the API breaks:
- https://github.com/vitest-dev/vitest/issues/3125
- https://github.com/vitest-dev/vitest/issues/3118

Reply:
Yeah, not intentional. Should've either not remove it or update `peerDependencies`. The next version is locked to be 0.30.0 though, so we will have to wait for a fix, but PR is welcome.